src/vsr/client.zig

```diff
@@ -19,21 +19,130 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
         pub const StateMachine = StateMachine_;
 
         pub const Error = error{
+            BatchSizeZero,
+            TooMuchData,
             TooManyOutstandingRequests,
```

Comment:
Let's use our most-significant bit ordering here for these names, e.g. `RequestLimitExceeeded` or `RequestBodyExceeded` or variants on that style.

Reply:
Not that my suggestions are what we should go with, but just to give an example of the MSB order.